python/metatensor-operations/metatensor/operations/manipulate_dimension.py

```diff
@@ -19,14 +19,14 @@
 from typing import List, Union
 
 from . import _dispatch
-from ._backend import Array, TensorBlock, TensorMap, torch_jit_script
+from ._backend import Array, Labels, TensorBlock, TensorMap, torch_jit_script
 
 
 def _check_axis(axis: str):
-    if axis not in ["keys", "samples", "properties"]:
+    if axis not in ["keys", "samples", "properties", "components"]:
```

Comment:
Very minor but I think the common order is "keys", "samples", "components", "properties"

```suggestion
    if axis not in ["keys", "samples", "components", "properties"]:
```